resources/styles/components/learning-guide.less

```diff
@@ -0,0 +1,239 @@
+@guide-width: 1270px;
+@guide-height: 390px;
+
+.learning-guide {
+  width: 100%;
+  background-repeat: repeat;
+  position: relative;
+
+  .x-axis-bg-repeat {
+    // global body padding-left/right prevents line from reaching ends at the moment...body classes with component name?
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 41px;
+    background: @tutor-guide-green;
+    z-index: -1;
+  }
+  .learning-guide-chart {
+    width: @guide-width;
+    height: @guide-height;
+    margin: 0 auto;
+
+    svg {
+      width: @guide-width;
+      height: @guide-height;
+
+      .main-title {
+        font-size: .13em;
+        font-family: Georgia;
+        font-weight: bold;
+        fill: #333333;
+
+        .show-course {
+          .no-select();
+          cursor: pointer;
+          fill: #1D9FDC;
+          font-size: .8em;
+        }
+      }
+      .circles {
+        .ace {
+          fill: @tutor-tertiary;
+        }
+        .cruising {
+          fill: @tutor-guide-green;
+        }
+        .too-low {
+          fill: @tutor-quaternary;
+        }
+        .grounded {
+          fill: @tutor-neutral-darker;
+        }
+        text {
+          // the unscaled resolution of the svg is only 100px wide, // so 2px is 2% of the width of the chart, i.e. fairly large
+          font-size: 0.119em;
+          fill: white;
+        }
+        circle {
+          stroke: white;
+          stroke-width: .25;
+        }
+        g {
+          cursor: pointer;
+
+          // &:hover {
+            opacity: 0.8;
+          }
+        }
+      }
+      .grid-lines {
+        line {
+          stroke: lightGrey;
+          stroke-width: 0.1;
+          // stroke-dasharray: 1 2;
+        }
+      }
+      .plot-lines {
+        line {
+          stroke: #FFF;
+          stroke-width: 0.4;
+          stroke-dasharray: 1.2 0.5;
+        }
+      }
+      .background-gradient {
+        // despite the attribute being named "stop-color" this sets the starting color
+        .start {
+          stop-color: #c1d8e5;
+        }
+        .end {
+          stop-color: white;
+        }
+      }
+      .foreground-hills {
+        fill: #66A333;
+      }
+      .background-hills {
+        fill: #83B553;
+      }
+      .x-axis {
+        text {
+          font-size: 0.13em;
+          fill: white;
+          font-family: Palatino;
```

Comment:
Actually, `Palatino` is not defined in `typography.less` (or in https://github.com/openstax/tutor-js/tree/master/resources/fonts). Is there a better one to use? maybe @Fredasaurus ?